core/mgmt/v1beta/mgmt_public_service.proto

```diff
@@ -64,6 +64,28 @@ service MgmtPublicService {
     option (google.api.method_visibility).restriction = "INTERNAL";
   }
 
+  // Get the authenticated user
+  //
+  // Accesses the authenticated user.
+  rpc GetAuthenticatedUser(GetAuthenticatedUserRequest) returns (GetAuthenticatedUserResponse) {
+    option (google.api.http) = {get: "/v1beta/user"};
```

Comment:
`GET /users/me` is part of `GET /users/<user_id>`. 
For `GET /users/<user_id>`, it will be an API that can be requested by anyone and should only contain public data.
However, for `/users/me`, it will contain more private data. To maintain consistency in the API response format when the requested `user_id` is changed, IMO, it is better to separate them into different API routes.

- `GET /users/<user_id>`: can be accessed by any user and only contains public data.
- `GET /user`: retrieves all data of the authenticated user.

Reference:
- Github API follows a similar [design](https://docs.github.com/en/rest/users/users?apiVersion=2022-11-28#get-the-authenticated-user).

Reply:
Clear, thanks for the references 👍 I hadn't seen that the `GetUser` endpoint was kept and that there was no previous `GET /users/me`, so I hadn't thought about the conflicting paths. With the documentation around the endpoint this distinction will be clear so this is valid ✅